plugins/rendermax/renderer_light.hpp

```diff
@@ -263,7 +265,7 @@ class lightThread
     void work(); //main light calculation function
     void combine(); //combine existing canvas into global lightmap
 public:
-    tthread::thread *myThread;
+    std::thread *myThread;
     bool isDone; //no mutex, because bool is atomic
```

Comment:
i know it's "widely believed" that "bool is always atomic" but the standard makes no such guarantee

see also https://stackoverflow.com/questions/16320838/when-do-i-really-need-to-use-atomicbool-instead-of-bool

Reply:
I don't think a mutex is needed for this? isDone can be an std::atomic<bool> from what I see.

I don't even know what rendermax even does lol.

Reply:
I agree with you. It should be an atomic if the value is shared between threads.